community-impact.html

```diff
@@ -161,52 +161,6 @@ <h3>Create Ongoing Engagement</h3>
     Add Community Impact to My Program
   </button>
 
-  <footer>
-    <div class="container footer-grid">
-      <div class="footer-section footer-company">
-        <h3>StayVisible</h3>
-        <p>P.O. Box 8234<br>New Fairfield, CT 06812<br>(888) 746-2111</p>
-      </div>
-      <div class="footer-section footer-links">
-        <h3>Quick Links</h3>
-        <ul>
-          <li><a href="#" onclick="document.querySelector('#search-form-header input[type=search]').focus(); return false;">Search</a></li>
-          <li><a href="contact.html">Contact Us</a></li>
-          <li><a href="contact.html">Support</a></li>
-          <li><a href="contact.html">Feedback and Suggestions</a></li>
-        </ul>
-      </div>
-      <div class="footer-section footer-social">
-        <h3>Follow Us</h3>
-        <div class="social-icons">
-          <a href="https://www.facebook.com/stayvisiblellc" target="_blank" aria-label="Facebook"><i class="fab fa-facebook-f"></i></a>
-          <a href="https://x.com/StayVizLLC" target="_blank" aria-label="X (Twitter)"><i class="fab fa-x-twitter"></i></a>
-          <a href="https://www.instagram.com/stayvisible2018/" target="_blank" aria-label="Instagram"><i class="fab fa-instagram"></i></a>
-          <a href="https://www.linkedin.com/company/stay-visible-llc/" target="_blank" aria-label="LinkedIn"><i class="fab fa-linkedin-in"></i></a>
-        </div>
-      </div>
-    </div>
-    <div class="container copyright">
-      <p>&copy; <span id="current-year"></span> StayVisible Promotions. All rights reserved.</p>
-    </div>
-  </footer>
-
-  <!-- Product Modal -->
```

Comment:
can you check why it is removing this modal? i don't think this was intended? We might need to add this back to this file and then merge this branch.